src/main/java/io/cryostat/net/web/http/api/v2/graph/GraphModule.java

```diff
@@ -93,26 +102,8 @@ public abstract class GraphModule {
     @Provides
     @Singleton
     static GraphQL provideGraphQL(
-            NodeTypeResolver nodeTypeResolver,
-            RecordingTypeResolver recordingTypeResolver,
-            RootNodeFetcher rootNodeFetcher,
-            EnvironmentNodesFetcher environmentNodesFetcher,
-            TargetNodesFetcher targetNodesFetcher,
```

Comment:
I just realized something, why doesn't `EnvironementNodeRecurseFetcher` show up here? Is it being used somewhere else somehow?

Reply:
I guess this was previously used for some earlier query that ended up removed. It doesn't look like it is actually used anymore at all. The implementation does something like `descendantTargets`, but it would be `descendantEnvironments` instead which doesn't seem particularly useful. I'll remove it.